llvm/include/llvm/BinaryFormat/ELFRelocs/V810.def

```diff
@@ -2,15 +2,21 @@
 #error "ELF_RELOC must be defined"
 #endif
 
+/* binutils 2.10-compatible offsets */
 ELF_RELOC(R_V810_NONE,      0)
-ELF_RELOC(R_V810_8,         1)
-ELF_RELOC(R_V810_16,        2)
-ELF_RELOC(R_V810_32,        3)
-ELF_RELOC(R_V810_DISP8,     4)
-ELF_RELOC(R_V810_DISP16,    5)
-ELF_RELOC(R_V810_DISP32,    6)
-ELF_RELOC(R_V810_LO,        7)
-ELF_RELOC(R_V810_HI,        8)
+ELF_RELOC(R_V810_9_PCREL,   1)
+ELF_RELOC(R_V810_26_PCREL,  2)
+ELF_RELOC(R_V810_HI_S,      3)
+ELF_RELOC(R_V810_HI,        4)
+ELF_RELOC(R_V810_LO,        5)
+ELF_RELOC(R_V810_32,        6)
+ELF_RELOC(R_V810_16,        7)
+ELF_RELOC(R_V810_8,         8)
 ELF_RELOC(R_V810_SDAOFF,    9)
-ELF_RELOC(R_V810_9_PCREL,   10)
-ELF_RELOC(R_V810_26_PCREL,  11)
\ No newline at end of file
+ELF_RELOC(R_V810_ZDAOFF,    10)
+ELF_RELOC(R_V810_TDAOFF,    11)
+
+/* LLVM additions. TODO: Are these necessary? */
+ELF_RELOC(R_V810_DISP8,     15)
+ELF_RELOC(R_V810_DISP16,    16)
+ELF_RELOC(R_V810_DISP32,    17)
```

Comment:
Opened #15 about it. These are probably unnecessary; I didn't know what PIE was when I started this, and went down a rabbit hole trying to produce a "position independent" rom